daemon.pro

```diff
@@ -1,7 +1,7 @@
 QT       += core network websockets widgets
 QT       -= gui
 
-#Wee need that for qwinoverlappedionotifier class which is private
+#We need that for qwinoverlappedionotifier class which is private
 win32: QT += core-private
```

Comment:
Please put this into an else {} block after the greaterThan(...)

We don't need core-private anymore in qt5.10

Reply:
qwinoverlappedionotifier.h uses <QtCore/private/qglobal_p.h>. 
qwinoverlappedionotifier.cpp uses <private/qobject_p.h> and <private/qiodevice_p.h>

So we still need core-private, or explicitly include these three files and any private files they need,

Reply:
Good point, it's ok for me then.